lib/fluent/supervisor.rb

```diff
@@ -1090,6 +1095,10 @@ def reload_config
             type: @config_file_type,
           )
 
+          build_additional_configuration do |additional_conf|
+            @conf += additional_conf if additional_conf
```

Comment:
```suggestion
            conf += additional_conf if additional_conf
```

This variable should be local here.

Reply:
Oh, good catch.